### ISSUE TITLE
Why use 5 dropouts?

### ISSUE THREAD

**Issue by minhhieutran2112** [abhishekkrthakur/long-text-token-classification#3]
Hi ya,

Thanks for your great repo. Can I ask something about the forward method of the model?

In the forward method, I can see that you are passing the transformer output through the classification head with 5 different dropout layer and averaging the logits & the loss to get the final output. Can I ask why? 

I also tried using only 1 dropout without these auxiliary loss but the result is substantially worse. Can I ask if you have any idea why this is the case?

Thank you

**Comment by wilmerhenao** [comment]
The idea comes from this paper: https://arxiv.org/abs/1905.09788

**Comment by minhhieutran2112** [comment]
Thank you!
